lib/logtail-rails/log_entry.rb

```diff
@@ -0,0 +1,13 @@
+module Logtail
+  class LogEntry
+    alias to_hash_unfiltered to_hash
+
+    def to_hash(options = {})
+      hash = to_hash_unfiltered(options)
+
+      parameter_filter = ActiveSupport::ParameterFilter.new(Rails.application.config.filter_parameters)
```

Comment:
I am a bit worried about performance here. There can be many `LogEntry` instances, therefore many redundant instantiations of `ParameterFilter`.

I thought of using a class variable `@@parameter_filter ||= ActiveSupport::ParameterFilter.new` to avoid it, but if there was a log entry during initialization, possibly not all `Rails.application.config.filter_parameters` would be configured during the first `to_hash` execution.

At that point, I didn't want to over engineer it for sake of premature optimization. WDYT?

Reply:
I would keep this as is. We can optimize later if anyone brings up performance issues.